hack.club.yaml

```
@@ -5,3 +5,7 @@
 hack.club:
   - type: ALIAS
     value: cname.vercel-dns.com.
+syria:
+  - ttl: 600
+    type: CNAME
+    value: hackclub-syria.github.io/website/.
```

DNS records can't have paths in them:
```suggestion
    value: hackclub-syria.github.io.
```

Additionally, since this is a Hack Club, please move this to hackclub.com!